### ISSUE TITLE
Ghosthawk can be used to remove stun(s)

### ISSUE THREAD

**Issue by StVelten** [keyteki/keyteki#1078]
Ghosthawk: Deploy. (This creature can enter play anywhere in your battleline.) Play: You may reap with each neighboring creature, one at a time.) 
If you place or deploy it next to stunned unit the game removes the stun. It should do nothing to exhausted and stunned units.

Screenshot (reproduced): 
https://imgur.com/a/BL4VhFW

Log (of the reproduced problem):

St-Velten is shuffling their deck
St-Velten is playing as the Archon: Othello Schöpf, Fischer der Unnahbarkeit
 St-Velten has connected to the game server
St-Velten draws 6 cards  up to their maximum hand size of 6
Key phase - St-Velten
St-Velten does not forge a key. They have 0 amber. The current cost is 6 Æmber 
House phase - St-Velten
St-Velten chooses untamed  as their active house this turn
Main phase - St-Velten
St-Velten plays Ghosthawk
 St-Velten is attempting to switch manual mode on
 St-Velten switches manual mode on
 St-Velten manually moves Tantadlin from their hand to their play area
 St-Velten stuns Ghosthawk
 St-Velten stuns Tantadlin
 St-Velten readies Tantadlin
 St-Velten readies Ghosthawk
 St-Velten switches manual mode off
Ready phase - St-Velten
St-Velten readies all of their cards 
Draw phase - St-Velten
St-Velten draws 1 cards  up to their maximum hand size of 6
End of turn 1
St-Velten: 0 Æmber 

Turn 2
Key phase - St-Velten
St-Velten does not forge a key. They have 0 amber. The current cost is 6 Æmber 
House phase - St-Velten
St-Velten chooses untamed  as their active house this turn
Main phase - St-Velten
St-Velten plays Ghosthawk
St-Velten uses Ghosthawk to reap with each of its neighbors in turn
St-Velten exhausts Ghosthawk to remove its stun
St-Velten exhausts Tantadlin to remove its stun

**Comment by romanogit** [comment]
Relates to #1076

**Comment by romanogit** [comment]
@StVelten 

Why do you think Ghosthawk should not unstun? How would it be different from Ganger Chieftain?

Ghosthawk:
`Play: You may reap with each neighboring creature, one at a time. `

Chieftain:
`Card Text | Play: You may ready and fight with a neighboring creature.`

**Comment by StVelten** [comment]
From 1.5 rules:
"A stunned, ready creature of the **active house** can be used by exhausting it to remove its stun status card."
 vs. (next in line) 
"If a card effect would cause a stunned creature to fight, reap or use an action or omni ability, instead that creature is exhausted and the stun status card is removed."

I had the same situation in a tournament a few months ago, where we got to the agreement it would not remove the stun. Take note that this was my first tournament and I did not know better.
For some reason I thought it would only work if the stunned creature is in the same house.
